lib/jekyll/commands/new.rb

```diff
@@ -7,7 +7,8 @@ class << self
         def init_with_program(prog)
           prog.command(:new) do |c|
             c.syntax "new PATH"
-            c.description "Creates a new Jekyll site scaffold in PATH"
+            c.description "Creates a new Jekyll site scaffold in PATH, and\n\t\t\t" \
```

Comment:
Its not done anywhere else in Jekyll. I added 3 tabs to keep it in-line with the first line. I agree, its a bit ugly.